bakerydemo/static/css/main.css

```diff
@@ -868,3 +1092,28 @@ span.outline {
     display: none;
   }
 }
+
+/* From Wagtail core */
+/* Responsive image/video classes */
+.rich-text img {
+    max-width: 100%;
+    height: auto;
+}
+.richtext-image.left{
+    float:left;
+}
+.richtext-image.right{
+    float:right;
+}
+.responsive-object {
+    position: relative;
+}
+.responsive-object iframe,
+.responsive-object object,
+.responsive-object embed {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+}
```

Comment:
Minor, but seems like you have a mix of 2-space and 4-space indentation in CSS. Personal pref, but my vote is 4-space indentation in CSS just to keep things simple and consistent with python.